src/CoreWCF.Primitives/src/CoreWCF/Security/SecuritySessionServerSettings.cs

```diff
@@ -1954,6 +1965,459 @@ public Task OpenAsync(CancellationToken token)
                     return Task.CompletedTask;
                 }
             }
+
+            
+        }
+
+        private class ServerSecurityDuplexSessionChannel : ServerSecuritySessionChannel //, IDuplexSessionChannel // Moving the 
+        {
```

Comment:
Moving the what?

Reply:
lol... moving the bridge :)... removed the comment, it was reference for me to go back do some work...